src/integration/blockchain/shared/util/blockchain-client.ts

```diff
@@ -10,5 +10,5 @@ export abstract class BlockchainClient {
   abstract getTokenBalances(assets: Asset[], address?: string): Promise<BlockchainTokenBalance[]>;
   abstract isTxComplete(txHash: string, confirmations?: number): Promise<boolean>;
   abstract getToken(asset: Asset): Promise<Currency>;
-  abstract sendSignedTransaction(tx: string): Promise<SignedTransactionResponse>;
+  abstract sendSignedTransaction(tx: string): Promise<EvmSignedTransactionResponse>;
```

Comment:
BtcClient also extends BlockchainClient but does not return EvmSignedTransactionResponse?